docs/quickstarts/insights-blueprints/insights-creating-blueprint-images.yml

```diff
@@ -0,0 +1,81 @@
+metadata:
+  name: create-blueprint
```

Comment:
The name of files and the name of the resources must match. There is a validation error in the pipeline:
```
19:00:11 #13 3.523 Help topic validation error. Error occured in:  docs/quickstarts/insights-blueprints/insights-blueprint.yaml
19:00:11 #13 3.525 panic: open docs/quickstarts/insights-blueprints/insights-blueprint.yaml: no such file or directory

```